js/react_native/android/src/main/java/ai/onnxruntime/reactnative/Readme.md

```diff
@@ -0,0 +1,7 @@
+NOTE: The raw resources contain a mix of .ORT and .ONNX format models.
+
+Originally the ORT React Native package used the ORT Mobile build, which only supports .ORT format models, and does not support the double or bool types.
+
+We're now using the 'full' ORT build in the React Native package, so both model formats and all types are supported.
+
+ONNX format models were added for the unsupported types.
```

Comment:
it's good to have this context.
I also added another readme related to these models [here](https://github.com/microsoft/onnxruntime/blob/a5d70d8609e6c35a2b75007cdd422beb09752e29/js/react_native/test_types_models.readme.md). not sure where the best place is, but maybe it would be good to have info about the models in one place?